src/test/e2e/helpers/database-init.ts

```diff
@@ -97,6 +98,7 @@ export default async function init(
         getLogger,
     });
 
+    log.setLogLevel('error');
```

Comment:
stop logging db migration messages in all tests

Reply:
Awesome! This was something I was planning to do yesterday :1st_place_medal: